nlp_tensorflow_streaming.ipynb

```diff
@@ -0,0 +1,1110 @@
+{
+  "cells": [
```

Comment:
Could you please move this to the `examples/notebooks/beam-ml/` folder? Otherwise LGTM